### ISSUE TITLE
As a user who browses the web with a screen reader, I want to load a IIIF resource into a viewer so I can explore the contents.

### ISSUE THREAD

**Issue by sdellis** [IIIF/iiif-stories#90]
# Description

The key idea is that this user will not be able to do view a discovered resource if a mouse is necessary to import it.

## Variation(s)

A similar use case would be for a tablet or mobile device, where one cannot drag from one browser window to another.

## Proposed Solutions

This is only a problem when the discovery client is different from the viewing client. One solution might be that the institution providing the discovery client could specify a default viewer (or multiple viewer options) for the resource so that "click to view" or "open in {viewer}" will work. 

## Additional Background

(more about your perspective, existing work, etc. goes here.)

**Comment by azaroth42** [comment]
Propose that it's not a screen-reader for limited sight audience, but a mobility impaired user that does not have access to fine grained mouse control, but has good vision.

**Comment by sdellis** [comment]
Yes, perhaps a slight revision to the use case will cover all three user types who cannot use a mouse for import: 1) vision impaired, 2) mobility impaired, and 3) mobile device user

Proposed revision:
> As a user who does not use a mouse, I want to load a IIIF resource into a viewer so I can explore the contents.

**Comment by mattmcgrattan** [comment]
Vatican 2017: Agreed to be in scope.
